### PR TITLE
Sorting

### PR DESCRIPTION
**Pull request by JoschaO**
This implements sorting in ticketlist. It also makes use of ngOnChanges for loading new lists. 
Status was removed as sortable option, because the first sort call to the backend would return an empty list. 
Sorry its late. It took tooo long to do. I should not be doing this at 2am.

### CODE REVIEW

File: TicketDesk/ngWebClient-StandAlone/src/app/app.component.html [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <base href="/">
+<title>Ticketeer</title>
```
Comment (kixiQu) [inline TicketDesk/ngWebClient-StandAlone/src/app/app.component.html]: Does this actually work?

File: TicketDesk/ngWebClient-StandAlone/src/app/ticket-center/ticket-center.component.ts [hunk 2]
```diff
@@ -18,7 +18,8 @@ export class TicketCenterComponent implements OnInit {
   ticketList: TicketStub[];
   pagination: {current: number, max: number } = {current: 1, max: 2};
   currentList;
-  listReady: Boolean = false;
+  sortingColumns: Set<String> = new Set();
```
Comment (kixiQu) [inline TicketDesk/ngWebClient-StandAlone/src/app/ticket-center/ticket-center.component.ts:21]: What's the advantage of using a set here?

File: TicketDesk/ngWebClient-StandAlone/src/app/app.component.html (hunk 1 above)
Comment (kixiQu) [reply TicketDesk/ngWebClient-StandAlone/src/app/app.component.html]: I should check it for myself But The VM

**Review by kixiQu — DISMISSED** [review]
I don't feel like I really understand this code. Under different circumstances, I would demand that we rectify this through more time studying it, explaining, refactoring, etc., but as it is....

File: TicketDesk/ngWebClient-StandAlone/src/app/ticket-center/ticket-center.component.ts (hunk 2 above)
Comment (JoschaO) [reply TicketDesk/ngWebClient-StandAlone/src/app/ticket-center/ticket-center.component.ts:21]: Set is so it only has unique values. We only care about the unique values for multisort. I could of done an array, and just checked before adding, but a set seemed easier.

**Review by kixiQu — APPROVED** [review]
te apruebo

**Pull request merged**